.github/workflows/pull-build-template-operator.yml

```diff
@@ -0,0 +1,35 @@
+name: pull-build-template-operator
+
+on:
+  push:
+    branches:
+      - main  # This will get tagged with `latest` and `v{{DATE}}-{{COMMIT_HASH_SHORT}}`
+  pull_request_target:
+    types: [ opened, edited, synchronize, reopened, ready_for_review ]
+
+permissions:
+  id-token: write # This is required for requesting the JWT token
+  contents: read # This is required for actions/checkout
+
+jobs:
+  compute-tag:
+    runs-on: ubuntu-latest
+    outputs:
+      tag: ${{ steps.get_tag.outputs.TAG }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Get the latest tag
+        id: get_tag
+        if: github.event_name == 'push'
+        run: echo "tag=latest" >> $GITHUB_OUTPUT
```

Comment:
let's not use this `latest`, it seems in the pipeline, it will automatically replace it with PR number, but this `latest` brings confusion, maybe just use a dummy example as mentioned [here](https://github.com/kyma-project/test-infra/blob/main/.github/workflows/pull-image-builder-test.yml#L26).

Reply:
The `latest` tag should be here. Only when there's a GitHub `push` event, meaning, every time there's a push to the `main` branch, the latest tag gets added. The control flow never reaches this line when its a Pull Request.

Moreover I tried it manually, the build job does not automatically add the `latest` label in the prod images, hence I added it here manually.

Reply:
ok, then it makes sense.